docs_user/modules/proc_using-new-subnet-ranges.adoc

```diff
@@ -1,43 +1,23 @@
 [id="using-new-subnet-ranges_{context}"]
 
-= Scenario 1: Using new subnet ranges
+= Configuring new subnet ranges
 
-This scenario is compatible with any existing subnet configuration, and can be
-used even when the existing cluster subnet ranges don't have enough free IP
-addresses for the new control plane services.
+You can define new IP ranges for control plane services that belong to a different subnet that is not used in the existing cluster. Then you configure link local IP routing between the old and new subnets to enable old and new service deployments to communicate. This involves using the {OpenStackPreviousInstaller} mechanism on a pre-adopted cluster to configure additional link local routes. This enables the EDP deployment to reach out to adopted nodes by using the old subnet addresses. You can use new subnet ranges with any existing subnet configuration, and when the existing cluster subnet ranges do not have enough free IP addresses for the new control plane services.
 
-The general idea here is to define new IP ranges for control plane services
-that belong to a different subnet that was not used in the existing cluster.
-Then, configure link local IP routing between the old and new subnets to allow
-old and new service deployments to communicate. This involves using {OpenStackPreviousInstaller}
-mechanism on pre-adopted cluster to configure additional link local routes
-there. This will allow EDP deployment to reach out to adopted nodes using their
-old subnet addresses.
+You must size the new subnet appropriately to accommodate the new control
+plane services. There are no specific requirements for the
+existing deployment allocation pools already consumed.
 
-The new subnet should be sized appropriately to accommodate the new control
-plane services, but otherwise doesn't have any specific requirements as to the
-existing deployment allocation pools already consumed. Actually, the
-requirements as to the size of the new subnet are lower than in the second
-scenario, as the old subnet ranges are kept for the adopted nodes, which means
-they don't consume any IP addresses from the new range.
+In the following procedure, you configure `NetworkAttachmentDefinition` custom resources (CRs) to use a different subnet from what is configured in the `NetConfig` CR for the same networks. The former range is used for control plane services, while the new range is used to manage IP Address Management (IPAM) for data plane nodes.
 
-In this scenario, you will configure `NetworkAttachmentDefinition` custom resources (CRs) to use a
-different subnet from what will be configured in `NetConfig` CR for the same
-networks. The former range will be used for control plane services,
-while the latter will be used to manage IPAM for data plane nodes.
+[IMPORTANT]
+Do not change adopted node IP addresses during the adoption process. List the IP addresses in the `fixedIP` fields in the `OpenStackDataPlaneNodeSet` CR `per-node` section.
```

Comment:
Is “per-node” the actual name of the section? I’m unable to find it in other guides, so I want to confirm.

Reply:
No, there is no `per-node`-named section.

An example of how this is configured can be traced e.g. here:

1. [The section with all nodes.](https://github.com/openstack-k8s-operators/openstack-operator/blob/b61660cac429b4c48a403d4bf9597c218782e722/config/samples/dataplane/networker/values.yaml#L18)
2. [Inside the section, an entry per node.](https://github.com/openstack-k8s-operators/openstack-operator/blob/b61660cac429b4c48a403d4bf9597c218782e722/config/samples/dataplane/networker/values.yaml#L19)
3. [Inside each entry, the field with the fixedIP.](https://github.com/openstack-k8s-operators/openstack-operator/blob/b61660cac429b4c48a403d4bf9597c218782e722/config/samples/dataplane/networker/values.yaml#L27)